.buildkite/pipeline.yml

```diff
@@ -0,0 +1,20 @@
+---
+steps:
+  - label: ":hammer: Test"
+    agents: queue=${AGENT_QUEUE}
```

Comment:
We should be able to define an `agents` entry in the Buildkite web-UI (YAML) configuration and remove the ones from here.